src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,79 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: Loader
+    Type: Loader
+    Threads: 10
+    Phases:
+      - Repeat: 1
+        Database: cold
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+      - Repeat: 1
+        Database: hot
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+      - Repeat: 1
+        Database: hot
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+      - Repeat: 1
+        Database: multithreaded
+        # Create 2 collections and populate with 5 threads / collection.
+        CollectionCount: 2
+        MultipleThreadsPerCollection: true
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+  - Name: MultipleLoadThreadsPerCollection
+    Type: Loader
+    Threads: 100
+    Phases:
+      - Repeat: 1
+        Database: multithreaded
+        # CollectionCount must be an even divisor of Threads.
+        # Phase config cannot contain Threads if MultipleThreadsPerCollection is true.
+        # Create 10 collections and populate with 10 threads per collection.
+        MultipleThreadsPerCollection: true
+        CollectionCount: 10
+        # DocumentCount is the total document count. Each thread populate an equal fraction of the count.
+        DocumentCount: 200000
```

Comment:
Shouldn't one of document count and batchsize be adjusted here? I think there are 20k docs per collection total, and, 10 threads per collection would give 2k docs per thread, but the batch size is 100k. I guess that means that it's one insert per thread, but still a bit confusing.